lib/off.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+module.exports = function off (state, options, eventName, handler) {
+  if (eventName.length > 0) {
```

Comment:
that is wrong. If eventName is set, you remove all listeners. But eventName is always set. Instead, you want to check if a handler was passed. If no handler was passed, remove all listeners, otherwise remove only one listener. 

The fact that all tests passed anyway is a sign that our test suite is not complete